### ISSUE TITLE
Suggestions and adjustments

### ISSUE THREAD

**Issue by silasprincipe** [iobis/manual#79]
- [x] [On section 3](https://manual-dev.obis.org/formatting.html): the phrase "At this step, there are different non arbitrary you need to do with your data, but it is important to determine which structure best suits your dataset before proceeding." sounds a little bit strange for me. Maybe rephrase?
- [x] [4.2.1.4 Step 2: Match with other registers](https://manual-dev.obis.org/name_matching.html): it's a bit confusing, because at first we say that we only accept three authorities (WORMS, NCBI and BOLD).
- [x] [4.2.3 Taxon Match Tools Overview](https://manual-dev.obis.org/name_matching.html#r-packages-for-taxon-matching): on table, where is "Outputs a tibble for each taxa name specified, Requires knowledge of R or python", change comma for a new line or ';'.
- [x] [4.5.1](https://manual-dev.obis.org/format_emof.html#what-data-goes-into-emof): convert note (_*For measurementTypeID, measurementUnitID, and measurementValueID you must use controlled ..._) into a quote format (using >).
- [x] [4.8](https://manual-dev.obis.org/common_formatissues.html#common-data-formatting-issues): suggestion - change 'Contents:' for some introductory phrase, e.g.: "When formatting data for submission on OBIS, some of the common problems you may find are: [then the list]"
- [x] [4.8 times/dates](https://manual-dev.obis.org/common_formatissues.html#temporal-dates-and-times): convert"A note about intervals..." to quoted format (with >)

The videos are great!

**Comment by pieterprovoost** [comment]
Added tasks, @silasprincipe would you mind creating separate issues for the suggestions?

**Comment by silasprincipe** [comment]
Sure, #80 and #81

**Comment by EliLawrence** [comment]
Thanks Silas! I've addressed all the suggestions here
